cmake/modules/Findcppcheck.cmake

```diff
@@ -19,7 +19,12 @@ function(FindCppcheck)
         PATHS "${OSQUERY_TOOLCHAIN_SYSROOT}"
       )
     endif()
-
+  elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
+    set(executable_name "cppcheck")
+    
+    set(optional_path_suffix_list 
+      PATH_SUFFIXES usr/local/bin
```

Comment:
This doesn't need a leading `/` ?

Reply:
It looks right. They're path _suffixes_ (I'm following the similar line for Linux, also I tested this and it works for both Linux and macOS)